### PR TITLE
aws-configure: add page

### PR DESCRIPTION
**Pull request by piraces**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.7.25

Related: #5653

### CODE REVIEW

File: pages/common/aws-configure.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Configure AWS CLI interactively (new configuration or updating the default):
+
+`aws configure`
+
+- Configure a named profile for AWS CLI interactively (new profile or updating the existing):
+
+`aws configure --profile {{production}}`
+
+- Get a configuration value from the configuration file:
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:14]:
```suggestion
- Display the value from a specific configuration variable:
```

File: pages/common/aws-configure.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure get {{region}}`
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:16]:
```suggestion
`aws configure get {{name}}`
```

File: pages/common/aws-configure.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Get a configuration value from a profile configuration file:
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:18]:
```suggestion
- Display the value for a configuration variable in a specific profile:
```

File: pages/common/aws-configure.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure get {{region}} --profile {{production}}`
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:20]:
```suggestion
`aws configure get {{name}} --profile {{profile_name}}`
```

File: pages/common/aws-configure.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Set a configuration value from the configuration file:
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:22]:
```suggestion
- Set the value of a specific configuration variable:
```

File: pages/common/aws-configure.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure set {{region}} {{us-west-1}}`
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:24]:
```suggestion
`aws configure set {{name}} {{value}}`
```

File: pages/common/aws-configure.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Set a configuration value from a profile configuration file:
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:26]:
```suggestion
- Set the value of a configuration variable in a specific profile:
```

File: pages/common/aws-configure.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure set {{region}} {{us-west-1}} --profile {{production}}`
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:28]:
```suggestion
`aws configure set {{name}} {{value}} --profile {{profile_name}}`
```

File: pages/common/aws-configure.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- List the AWS CLI configuration data:
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:30]:
```suggestion
- List the configuration entries:
```

File: pages/common/aws-configure.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure list`
+
+- List the AWS CLI profile configuration data:
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:34]:
```suggestion
- List the configuration entries for a specific profile:
```

File: pages/common/aws-configure.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+`aws configure list --profile {{production}}`
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:36]:
```suggestion
`aws configure list --profile {{profile_name}}`
```

File: pages/common/aws-configure.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
+# aws configure
+
+> Configure AWS CLI options.
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:3]:
```suggestion
> Manage configuration for the AWS CLI.
```

File: pages/common/aws-configure.md (hunk 2 above)
Comment (piraces) [reply pages/common/aws-configure.md:16]: Ups.. did't notice that. Sorry.

**Comment by piraces** [comment]
Thanks for the work correcting this PR changes. Will learn that for future contributions...

File: pages/common/aws-configure.md [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+> Manage configuration for the AWS CLI.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/configure/>.
+
+- Configure AWS CLI interactively (creates a new configuration or updates the default):
+
+`aws configure`
+
+- Configure a named profile for AWS CLI interactively (creates a new profile or updates an existing one):
+
+`aws configure --profile {{production}}`
```
Comment (navarroaxel) [inline pages/common/aws-configure.md:12]:
```suggestion
`aws configure --profile {{profile_name}}`
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

> Wow, that's a lot of amazing reviews, Great job navarroaxel.

**Pull request merged**